silk/node/wpantund_base.py

```diff
@@ -128,9 +128,17 @@ def un_allowlist_node(self, item):
     #   wpan_node functionality
     #################################
 
-    def form(self, network, role, xpanid=None, panid=None, mesh_local_prefix=None):
+    def form(self, network, role, xpanid=None, panid=None, mesh_local_prefix=None, add_ip_addr="Yes"):
         """
         Form the PAN specified by the arguments.
+        Args:
```

Comment:
Please use this format for consistency.

Args:
            target (str): target address.

Reply:
Ack. Done